lint-staged.config.mjs

```diff
@@ -1,12 +1,16 @@
 const path = require('node:path');
 
+/**
+ * @param {string[]} filenames
+ */
+
```

Comment:
In this file ESLint threw an TS error

```
Invalid type "any" of template literal expression.eslint[@typescript-eslint/restrict-template-expressions
```

To fix this error, I added a JSDoc comment, so TS understood the type of `filenames` used as a parameter in the `buildEslintCommand` function.